vault/diagnose/tls_test.go

```diff
@@ -132,15 +87,183 @@ func TestTLSTrailingData(t *testing.T) {
 	}
 }
 
-func TestTLSExpiredCert(t *testing.T) {
+func TestTLSExpiredCert(t *testing.T) {}
```

Comment:
what is this intended to do?

Reply:
This makes sure an expired cert is caught with the appropriate error message. Filled in now! I had to wait a day for my openssl certificate to expire as openssl only allows > day expiration times on certs.

Reply:
oh cool :)